app/schema/order_status.py

```diff
@@ -0,0 +1,7 @@
+import enum
+
+
+class Status(enum.Enum):
+    IN_PROGRESS: str = "IN PROGRESS"
+    DONE: str = "DONE"
+    CANCELED: str = "CANCELED"
```

Comment:
```python
class Status(enum.Enum):
    NEW = "NEW"
    IN_PROGRESS = "IN PROGRESS"
    DONE = "DONE"
    CANCELED = "CANCELED"

Reply:
first value - NEW - is default value
IN_PROGRESS will be set by bot when it be taken in process